builds/e2e/templates/e2e-setup.yaml

```diff
@@ -117,6 +151,46 @@ steps:
       $edgeHubImage = "$imagePrefix-hub:$imageTag";
     }
 
+    if ('$(enableManifestSigning)' -eq 'true')
+    {  
+      $testManifestSigningGoodRootCaPath = Convert-Path "$(manifestSigningCertDir)/good_root_ca.pem";
+      $testManifestSigningBadRootCaPath = Convert-Path "$(manifestSigningCertDir)/bad_root_ca.pem";
+      $testManifestSigningIntermediateCaPath = Convert-Path "$(manifestSigningCertDir)/intermediate_ca.pem";
+      $testManifestSigningSignerCertPath = Convert-Path "$(manifestSigningCertDir)/signer_cert.pem";
+      $testManifestSigningSignerKeyPath = Convert-Path "$(manifestSigningCertDir)/signer_key.key";
+      $testManifestSigningDeploymentPath = Convert-Path "$(manifestSigningDeploymentDir)/deployment.json";
+      $testManifestSigningSignedDeploymentPath = Convert-Path "$(manifestSigningDeploymentDir)/signed_deployment.json";
+      $testManifestSignerClientDirectory = Convert-Path "$(manifestSignerClientDir)";
+      $testManifestSignerClientProjectPath = Convert-Path "$(manifestSignerClientDir)/ManifestSignerClient.csproj";
+      $testManifestSigningLaunchSettingsPath = Convert-Path "$(manifestSignerClientDir)/Properties/launchSettings.json";
+
+      $testManifestSigningDefaultLaunchSettings = @{
+        profiles = @{
+          ManifestSignerClient = @{
+            commandName = "Project";
+            environmentVariables = @{  
+              USE_TESTING_CA = "true";
+              DEPLOYMENT_MANIFEST_FILE_PATH = "$testManifestSigningDeploymentPath";
+              SIGNED_DEPLOYMENT_MANIFEST_FILE_PATH = "$testManifestSigningSignedDeploymentPath";
+              MANIFEST_TRUST_INTERMEDIATE_CA_PATH = "$testManifestSigningIntermediateCaPath";
+              MANIFEST_TRUST_SIGNER_PRIVATE_KEY_PATH = "$testManifestSigningSignerKeyPath";
+              MANIFEST_TRUST_SIGNER_CERT_PATH = "$testManifestSigningSignerCertPath";
+              DSA_ALGORITHM = "ES256";
+            }
+          }
+        }
+      };
+
+      $testManifestSigningDefaultLaunchSettings = $testManifestSigningDefaultLaunchSettings | ConvertTo-Json -Depth 3 
+      ##$testManifestSigningDefaultLaunchSettings | ConvertTo-Json -Depth 3 | Out-File -Encoding Utf8 "$testManifestSigningLaunchSettingsPath"
+      $testManifestSigningDefaultLaunchSettings | Out-File -Encoding Utf8 "$testManifestSigningLaunchSettingsPath"
+      echo "LaunchSettingPath is"
+      Get-Content -Path "$testManifestSigningLaunchSettingsPath"
+      Get-Content -Path "$testManifestSigningSignerCertPath"
```

Comment:
Some cleanup is required here, but you might want to see the launch settings if we have to debug this test later.